### ISSUE TITLE
IANA Port Reservation for OTLP HTTP

### ISSUE THREAD

**Issue by mtwo** [open-telemetry/community#2430]
OTLP has an official IANA reservation for port 4317 for OTLP gRPC traffic. We've had a long-standing desire to reserve port 4318 for OTLP HTTP traffic, as this is the convention that OpenTelemetry's OTLP implementations use by default.

When we applied for port 4318 in 2021, IANA's response was "This should be rejected. GRPC and REST interfaces can be supported on the same port and there are plenty of examples of how to do this on the web.". They are correct, however the fact remains that most OTLP HTTP traffic is taking place on port 4318.

I have volunteered to re-open this request with IANA, however I we'll need to respond to their statement. Beyond stating that OTLP is incredibly popular and that it already uses port 4318 for HTTP traffic, are there any other arguments that we can use to persuade them?

@tigrannajaryan @atoulme FYI

**Comment by tigrannajaryan** [comment]
> I have volunteered to re-open this request with IANA, however I we'll need to respond to their statement. Beyond stating that OTLP is incredibly popular and that it already uses port 4318 for HTTP traffic, are there any other arguments that we can use to persuade them?

Yes, with where we are right now there is no way we will change port and break so many existing deployments of OTLP. We are way beyond the point of no return on that. _Maybe_ we could devise some graceful transition method but I don't think the effort is worth it.

**Comment by mtwo** [comment]
Agreed, the way that I see it is that we're going to use 4318 regardless, but we'd still like to get the official registration for it.

**Comment by svrnm** [comment]
While I agree that we are beyond a point of return for using 4318, I am worried that the response will still be the same, like "we told you in 2021, so why would we change our mind in 2024"? 

What other good arguments can we bring that separating ports is a good idea? Are there any security reasons we can give? 

I am curious to hear @open-telemetry/collector-maintainers thoughts on that.

**Comment by mtwo** [comment]
Closing this: while we will continue to use 4318, we have nothing new to present to IANA during an appeal. Please comment / re-open if you disagree.
